### DISCUSSION TITLE
fixes for version 7 CVEs

### DISCUSSION THREAD

**Discussion by dotlambda** [ImageMagick/ImageMagick6#135] [Q&A]
NIST [lists](https://nvd.nist.gov/vuln/search/results?form_type=Advanced&results_type=overview&seach_type=all&query=cpe:2.3:a:imagemagick:imagemagick:6.9.12-1:*:*:*:*:*:*:*) many CVEs as affecting ImageMagick 6.9.12-1. These seem to be CVEs reported for ImageMagick 7. How can I check whether these apply to version 6 and in which release they were fixed?

**Comment by dlemstra** [comment]
Most CVE's also have a corresponding issue that you can find in the report. And if those issues have a commit that is done in the IM6 repository you know that it also affected ImageMagick 6.

**Reply by dotlambda** [reply]
Is there a changelog which lists the CVEs fixed in a given release of ImageMagick6?

**Reply by urban-warrior** [reply]
No, CVE's are submitted by ImageMagick users and they sometimes will associate the CVE with the original issue posted here.  The original issue posting will typically also include the patch we committed to resolve the issue.  Try searching for `CVE` in the open and closed issues.
